var/spack/repos/builtin/packages/celeritas/package.py

```diff
@@ -149,4 +153,12 @@ def cmake_args(self):
             # JSON is required for 0.5 and later
             args.append(define("CELERITAS_USE_JSON", True))
 
+        if self.version >= Version("0.6"):
```

Comment:
Avoid creating a Version object.
```suggestion
        if self.spec.satisfies("0.6:"):
```